tests/test_file.py

```diff
@@ -177,7 +177,7 @@ def test_token_refresh_good_store(self):
         new_cred.access_token = 'bar'
         s.put(new_cred)
 
-        credentials._refresh(lambda x: x)
+        credentials._refresh(None)
```

Comment:
The `good_store` part of `test_token_refresh_good_store` means the refresh method is not needed, so `coverage` was (correctly) flagging `lambda x: x` as something which was never executed.